public/index.html

```diff
@@ -18,6 +18,7 @@ <h1>Entropia</h1>
       </div>
 
       <div class="club-status-wrapper">
+        <a href="https://mapall.space/heatmap/show.php?id=Entropia" target="_blank" rel="noopener noreferrer"></a>
         <span class="club-status"></span>
```

Comment:
Ich frag mal ganz naiv, sollte sich das </a> nicht eher nach dem `<span class="club-status"></span>` Schließen?

Reply:
Disclaimer, wenig Ahnung von HTML. Daher naiv zurückgefragt; wieso? Bzw kommt drauf an, so viel ich weiß?

Reply:
Man möchte ja den Teil den man anklickt innerhalb des ``<a>`` tags haben. Und das ist ja eigentlich diese Club Status anzeige...

Beispiel:
```html
<a href="http://example.com">Dieser Text führt nach example.com</a>
```

Reply:
Vielleicht ergänzend ein bisschen Kontext:

`<span class=club-status"></span>` ist das Element, in welches per Javascript nach Abfrage der SpaceAPI das Wort "geöffnet" bzw. "geschlossen" geschrieben wird und welches, ebenfalls per Javascript, eine zusätzliche Klasse erhält, um die grüne oder rote Hintergrundfarbe zu setzen.

Es könnte also auch das `<span>` durch den vorgeschlagenen `<a>`-Tag mit der Klasse `club-status` ersetzt werden.